src/main/java/com/ordina/surfforecast/web/BeachController.java

```diff
@@ -1,59 +1,67 @@
-                                                    throws RecordNotFoundException {
-        System.out.println(beach);
-        BeachEntity updated = service.createOrUpdateBeach(beach);
-        return new ResponseEntity<BeachEntity>(updated, new HttpHeaders(), HttpStatus.OK);
-    }
-
-    @CrossOrigin(origins = {"https://surf-forecast.herokuapp.com", "http://localhost:3000"})
-    @DeleteMapping("/{id}")
-    public HttpStatus deleteBeachById(@PathVariable("id") Long id)
-                                                    throws RecordNotFoundException {
-        service.deleteBeachById(id);
-        return HttpStatus.FORBIDDEN;
-    }
- 
+package com.ordina.surfforecast.web;
+
+import java.net.URISyntaxException;
+import java.util.List;
+
+import com.ordina.surfforecast.exception.RecordNotFoundException;
+import com.ordina.surfforecast.service.BeachService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpHeaders;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+import com.ordina.surfforecast.model.BeachEntity;
+
+
+@RestController
+@CrossOrigin(origins = {"https://surf-forecast.herokuapp.com", "http://localhost:3000"})
+@RequestMapping("/beaches")
+public class BeachController
+{
+    @Autowired
+    BeachService service;
+
+    @CrossOrigin(origins = {"https://surf-forecast.herokuapp.com", "http://localhost:3000"})
+    @GetMapping
+    public ResponseEntity<List<BeachEntity>> getAllBeaches() throws URISyntaxException {
+        List<BeachEntity> list = service.getAllBeaches();
+ 
+        return new ResponseEntity<List<BeachEntity>>(list, new HttpHeaders(), HttpStatus.OK);
+    }
+
+    @CrossOrigin(origins = {"https://surf-forecast.herokuapp.com", "http://localhost:3000"})
+    @GetMapping("/{id}")
+    public ResponseEntity<BeachEntity> getBeachById(@PathVariable("id") Long id)
+                                                    throws RecordNotFoundException {
+        BeachEntity entity = service.getBeachById(id);
+ 
+        return new ResponseEntity<BeachEntity>(entity, new HttpHeaders(), HttpStatus.OK);
+    }
+
+    @CrossOrigin(origins = {"https://surf-forecast.herokuapp.com", "http://localhost:3000"})
+    @PostMapping
+    public ResponseEntity<BeachEntity> createBeach(@RequestBody BeachEntity beach)
+                                                    throws RecordNotFoundException {
+        System.out.println(beach);
+        BeachEntity updated = service.createOrUpdateBeach(beach);
+        return new ResponseEntity<BeachEntity>(updated, new HttpHeaders(), HttpStatus.OK);
+    }
+
+    @CrossOrigin(origins = {"https://surf-forecast.herokuapp.com", "http://localhost:3000"})
+    @PutMapping("/{id}")
+    public ResponseEntity<BeachEntity> updateBeach(@RequestBody BeachEntity beach) {
+        System.out.println(beach);
```

Comment:
We can delete the System.out.println here

Reply:
Done!